### DISCUSSION TITLE
Lossy Cutting

### DISCUSSION THREAD

**Discussion by Jay123210599** [mifi/lossless-cut#2047] [Q&A]
Can LosslessCut do lossy cutting? If not, then what tools can? I am looking for a video splitter where I can split videos into segments, burn subtitles into them, and make image sequences out of them.

**Comment by mifi** [comment]
https://github.com/mifi/lossless-cut/blob/master/issues.md#faq

As for other tools im not sure but you can try to ask this exact question to chatgpt
